deployment/scripts/.env

```diff
@@ -0,0 +1,30 @@
+# Kubernetes Provider Enviromment Variables
+k8s_provider="eks"
+
+# Helm Enviroment Variables
+fuel_core_image_repository="ghcr.io/fuellabs/fuel-core"
+fuel_core_image_tag="latest"
+fuel_core_pod_replicas="1"
+base64_github_auth_token="base64_your_github_auth_token"
+
```

Comment:
Should we have a config var for the k8s namespace?

Reply:
We can if you would like @Voxelot but the namespace is automatically created per helm deployment here: https://github.com/FuelLabs/fuel-core/blob/roy-fuel-eks-helm-charts/deployment/scripts/fuel-core-deploy.sh#:~:text=%2D%2Dcreate%2Dnamespace%20%5C,namespace%3Dfuel%2Dcore%20%5C